rust/windows-client/src-tauri/tauri.conf.json

```diff
@@ -0,0 +1,47 @@
+{
+  "build": {
+    "beforeDevCommand": "",
+    "beforeBuildCommand": "",
+    "devPath": "../src",
+    "distDir": "../src",
+    "withGlobalTauri": true
+  },
+  "package": {
+    "productName": "firezone-windows-client",
+    "version": "0.0.0"
```

Comment:
I just realized JSON is the one place we can't use `# mark:automatic-version` 😭 

That's fine for now